templates/case-studies/index.html

```diff
@@ -1,72 +1,100 @@
+                  hi_def=True,
+                  loading="lazy") | safe
+        }}
+      </div>
+    {% endif %}
+  {%- endcall -%}
   
   <!-- Text + list section -->
-  {% import "case-studies/templates/_macros-text-list.jinja" as text_list_template %}
-  {{ text_list_template.text_list(
-      title="About the European Space Agency (ESA)",
-      list_items=[
-        "Founded in 1975, ESA is a 22-member intergovernmental space exploration organisation.",
-        "The European Space Operations Centre (ESOC), one of ESA’s premises in Germany, is home to the engineering teams that control spacecraft in orbit, manage the global tracking station network, and design and build the systems on the ground that support missions in space.",
-        "ESOC is responsible for a wide range of IT infrastructure and systems to help fly these unique missions, having been responsible for flying more than 80 custom-built satellites since it was established."
-      ]
-    ) 
-  }}
+  {%- call(slot) text_list(title="About the European Space Agency (ESA)") -%}
+    {%- if slot == 'list_item_content_1' -%}
+      Founded in 1975, ESA is a 22-member intergovernmental space exploration organisation.
+    {%- endif -%}
+    {%- if slot == 'list_item_content_2' -%}
+    The European Space Operations Centre (ESOC), one of ESA’s premises in Germany, is home to the engineering teams that control spacecraft in orbit, manage the global tracking station network, and design and build the systems on the ground that support missions in space.
+    {%- endif -%}
+    {%- if slot == 'list_item_content_3' -%}
+    ESOC is responsible for a wide range of IT infrastructure and systems to help fly these unique missions, having been responsible for flying more than 80 custom-built satellites since it was established.
+    {%- endif -%}
+  {%- endcall -%}
 
   <!-- Highlights section -->
-  {% import "case-studies/templates/_macros-highlights.jinja" as highlights_template %}
-  {{ highlights_template.highlights(
-      list_items=[
-        "Architecture, design, and deployment of Canonical’s distributions of Ceph and Kubernetes.",
-        "Timely updates and 24/7 monitoring of missioncritical equipment.",
-        "ESA has operated more than 87 missions from ESOC, ranging from understanding our own planet to exploring our solar system and beyond to help advance scientific knowledge.",
-        "ESOC is responsible for a wide range of IT infrastructure and systems to help fly these unique missions, having been responsible for flying more than 80 custom-built satellites since it was established."
-      ]
-    ) 
-  }}
+  {%- call(slot) highlights(num_items=4) -%}
+    {%- if slot == 'highlights_item_content_1' -%}
+      <p>Architecture, design, and deployment of Canonical’s distributions of Ceph and Kubernetes.</p>
+    {%- endif -%}
+    {%- if slot == 'highlights_item_content_2' -%}
+      <p>Timely updates and 24/7 monitoring of missioncritical equipment.</p>
```

Comment:
Missing a space between "mission" and "critical"

```suggestion
      <p>Timely updates and 24/7 monitoring of mission critical equipment.</p>
```